src/ssl.c

```diff
@@ -3029,6 +3029,29 @@ void* wolfSSL_CTX_get_TicketEncCtx(WOLFSSL_CTX* ctx)
 
     return ctx->ticketEncCtx;
 }
+
+/* set the maximum number of tickets to send
+ * return WOLFSSL_SUCCESS on success and WOLFSSL_FAILURE on fail
+ */
+int wolfSSL_CTX_set_num_tickets(WOLFSSL_CTX* ctx, size_t mxTickets)
+{
+    if (ctx == NULL)
+        return WOLFSSL_FAILURE;
+
+    ctx->maxTicketTls13 = mxTickets;
```

Comment:
Should we set a "cap" on the `maxTicketTls13`? Seems like there is a reasonable limit we can restrict. Not to mention the incoming arg is `size_t`, but the storage variable is `byte`.